spikes/xml/claim.xml

```diff
@@ -0,0 +1,237 @@
+<claim>
+    <claim_details>
+        <claim_uuid>2161f858-cb16-4c77-9792-28defcbc5688</claim_uuid>
+        <!-- type can be Advocate, Litigator Final, Litigator Interim, Litigator Transfer -->
+        <type>Advocate</type>
+        <!--provider_key is the key that CCR/CCLF use to identify provider-->
+        <provider_key>2161f858-cb16-4c77-9792-28defcbc5688</provider_key>
+        <!--We can either send an id that identifies the creator to CCCD, or an id used byCCRR/CCLF -->
```

Comment:
Well this is the nub of what Phil Eades was talking about.  We need to agree a set of keys to identify each provider (and court, offence class, offence catetory etc).  What I'm trying to convey here, is that we send that key.